pkg/actors/actors.go

```diff
@@ -187,20 +155,25 @@ func newActorsWithClock(opts ActorsOpts, clock clock.WithTicker) Actors {
 	appHealthy := &atomic.Bool{}
 	appHealthy.Store(true)
 	ctx, cancel := context.WithCancel(context.Background())
-	return &actorsRuntime{
+
+	remindersProvider := reminders.NewRemindersProvider(clock, internal.RemindersProviderOpts{
+		StoreName: opts.StateStoreName,
+		Config:    opts.ActorsConfig.Config,
+	})
+
+	a := &actorsRuntime{
 		appChannel:           opts.AppChannel,
 		grpcConnectionFn:     opts.GRPCConnectionFn,
-		config:               opts.Config,
+		config:               opts.ActorsConfig,
+		coreConfig:           &opts.ActorsConfig.Config,
```

Comment:
If "coreConfig" is part of `opts.ActorConfig`, why do we need to store them separately in the object? Can't we just store `config: opts.ActorsConfig` and then access the `Config` property of that?

Reply:
Had placed it, as had to refer everywhere `config.Config`.
Removed it now as per the comment, and using `actorsConfig.Config`